x-pack/test_serverless/functional/config.base.ts

```diff
@@ -11,15 +11,17 @@ import { pageObjects } from './page_objects';
 import { services } from './services';
 import type { CreateTestConfigOptions } from '../shared/types';
 
-export function createTestConfig(options: CreateTestConfigOptions) {
+export function createTestConfig<TServices extends {} = typeof services>(
+  options: CreateTestConfigOptions<TServices>
+) {
   return async ({ readConfigFile }: FtrConfigProviderContext) => {
     const svlSharedConfig = await readConfigFile(require.resolve('../shared/config.base.ts'));
 
     return {
       ...svlSharedConfig.getAll(),
 
       pageObjects,
-      services,
+      services: { ...services, ...options.services },
```

Comment:
good solution, we shouldn't expose those `ebt` FTR services by default as it probably won't work against MKI